consensus/mint-client/src/config.rs

```diff
@@ -338,6 +338,40 @@ pub enum Commands {
         #[clap(long, env = "MC_MINTING_OUTPUT_JSON")]
         output_json: Option<PathBuf>,
     },
+
+    /// Load a previously-serialized file produced by this tool and print its
+    /// contents in a human-friendly way.
+    Dump {
+        /// The file to load
+        #[clap(long, parse(try_from_str = load_tx_file_from_path), env = "MC_MINTING_TX_FILE")]
+        tx_file: TxFile,
+    },
+
+    /// Sign a transaction file produced by this tool, rewriting the file with
+    /// the appended signature(s).
+    Sign {
+        /// The file to sign
+        #[clap(long, env = "MC_MINTING_TX_FILE")]
+        tx_file: PathBuf,
+
+        /// The key(s) to sign the transaction with.
+        #[clap(
+            long = "signing-key",
+            required_unless_present = "signatures",
+            parse(try_from_str = load_key_from_pem),
+            env = "MC_MINTING_SIGNING_KEY"
```

Comment:
This was updated in https://github.com/mobilecoinfoundation/mobilecoin/pull/2000

```suggestion
            env = "MC_MINTING_SIGNING_KEYS"
```